src/remix/remix-wizard.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  addSentryCliRc,
+  askForProjectSelection,
+  askForSelfHosted,
+  askForWizardLogin,
+  confirmContinueEvenThoughNoGitRepo,
+  ensurePackageIsInstalled,
+  getPackageDotJson,
+  installPackage,
+  isUsingTypeScript,
+  printWelcome,
+} from '../utils/clack-utils';
+import { hasPackageInstalled } from '../utils/package-json';
+import { WizardOptions } from '../utils/types';
+import {
+  initializeSentryOnEntryClient,
+  initializeSentryOnEntryServer,
+  updateBuildScript,
+  instrumentRootRoute,
+  isRemixV2,
+  loadRemixConfig,
+} from './sdk-setup';
+
+export async function runRemixWizard(options: WizardOptions): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Remix Wizard',
+    promoCode: options.promoCode,
+  });
+
+  await confirmContinueEvenThoughNoGitRepo();
+
+  const remixConfig = await loadRemixConfig();
+  const packageJson = await getPackageDotJson();
+  await ensurePackageIsInstalled(packageJson, '@remix-run/node', 'Remix');
```

Comment:
is this still used if they are using the generic `@remix-run/serve` lib?

Should we instead check for `@remix-run/dev`?

Reply:
Yes, good point 👍 Updated